docs/contracts/VotingMachines/README.md

```diff
@@ -1,11 +1,11 @@
 # VotingMachine
 
-***VotingMachine*** is a pluggable component, provided by DAOstack, which can be use by a DAO's schemes to manage a voting
+***VotingMachine*** is a pluggable component, provided by DAOstack, which can be used by a DAO's schemes to manage a voting
 process on a certain proposal.
 
-The VotingMachine maintains the voting process ,the proposals to vote on ,collect votes and determines the winning vote according to pre define DAO's configuration.
+The VotingMachine maintains the voting process and the proposals to vote on, collect votes, and determines the winning vote according to the DAO's pre-defined configuration.
 
-In some cases, such as GenesisProtocol, it also collect stakes, promote proposals and maintains a reputation flow for participants according to pre define DAO's configuration.  
+In some cases, such as GenesisProtocol, it also collect stakes, promotes proposals, and maintains a reputation flow for participants according to the DAO's pre-defined configuration.  
```

Comment:
In some cases, such as GenesisProtocol, it also collect**s** stakes, promotes proposals, and maintains a reputation flow for participants according to the DAO's pre-defined configuration.